src/app/common/drop-down-list/drop-down-list.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
```

Comment:
This component isn't used anywhere, do you need it?

Reply:
should I delete this component ?